lib/rubocop/cop/rspec/repeated_subject_call.rb

```diff
@@ -64,16 +70,11 @@ def on_top_level_group(node)
 
         private
 
-        def detect_offense(example_node, subject_node)
-          walker = subject_node
+        def detect_offense(subject_node)
+          subject_node.each_ancestor(:block) do |block|
+            next if subject_node.chained?
```

Comment:
I have never seen this method. Is it old enough for us to be sure it will be there with a minimum required rubocop and rubocop-ast versions we require?

Reply:
It was there like forever. I just wonder if we should not just skip the whole each_ancestor in this case, instead of skipping each iteration 😅

Reply:
> we should not just skip the whole each_ancestor in this case, instead of skipping each iteration 😅

Indeed! Thank you catching this.